docs/rst/tutorials/core/deployment/static_edp/static_edp.rst

```diff
@@ -0,0 +1,231 @@
+Whereas the endpoint profile has to be set using the ROS 2 Topic name, the static discovery configuration must used the DDS Topic name.
+
+Run again the talker-listener demo loading the corresponding XML configuration file using ``FASTRTPS_DEFAULT_PROFILES_FILE`` environment variable:
+
+.. tabs::
+
+    .. tab:: Shell 1 (Listener)
+
+        .. code-block:: bash
+
+            source /opt/vulcanexus/humble/setup.bash
+            FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_config_file>/listener_profile.xml ros2 run demo_nodes_cpp listener
+
+    .. tab:: Shell 2 (Talker)
+
+        .. code-block:: bash
+
+            source /opt/vulcanexus/humble/setup.bash
+            FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_config_file>/talker_profile.xml ros2 run demo_nodes_cpp talker
+
+If the traffic is captured again using Wireshark, this time no ``DATA(w)`` or ``DATA(r)`` is exchanged, but communication is established.
+
+.. raw:: html
+
+    <video width=100% height=auto autoplay loop controls muted>
+        <source src="../../../../../_static/resources/tutorials/core/deployment/static_edp/static_edp.mp4">
+        Your browser does not support the video tag.
+    </video>
+
+.. _tutorials_deployment_static_edp_fullfledged_discovery:
+
+Full-fledged Static EDP discovery
+---------------------------------
+
+ROS 2 provides several introspection tools like ``ros2 topic list`` or ``ros2 node list``.
+Running these commands with the previous example will shown only the default ROS 2 topics (``/parameter_events`` and ``/rosout``) and no node.
+The ``/chatter`` topic and the ``/talker`` and ``/listener`` nodes would not appear even though they are up and running and communicating.
+The issue is related to the internal endpoints that ROS 2 ecosystem, and by extension Vulcanexus, launch within each node to ensure these tools work as expected.
+Also, the vast majority of these tools depend on another DomainParticipant known as the ROS 2 Daemon.
+This section will complete the previous XML files so a full-fledged communication within Vulcanexus ecosystem is achieved, with working introspection tools.
+
+Below, the complete XML files are shown.
+Please, replaced the previous XML configuration files with these ones and the new one required to configure the ROS 2 daemon DomainParticipant.
+
+.. tabs::
+
+  .. tab:: talker_profile.xml
+
+    .. literalinclude:: /resources/tutorials/core/deployment/static_edp//complete_talker_profile.xml
+      :language: xml
+
+  .. tab:: listener_profile.xml
+
+    .. literalinclude:: /resources/tutorials/core/deployment/static_edp//complete_listener_profile.xml
+      :language: xml
+
+  .. tab:: ros2_daemon_profile.xml
+
+    .. literalinclude:: /resources/tutorials/core/deployment/static_edp//ros2_daemon_profile.xml
+      :language: xml
+
+  .. tab:: static_edp_info.xml
+
+    .. literalinclude:: /resources/tutorials/core/deployment/static_edp//complete_static_edp_info.xml
+      :language: xml
+
```

Comment:
I would add something here to clarify why all this new data readers and data writers were necessary to be included in the new xmls. It is mentioned the issue with ROS2 environment, but some clarification would be welcome